tests/functional/test_install_reqs.py

```diff
@@ -106,6 +107,37 @@ def test_multiple_requirements_files(script, tmpdir):
     assert script.venv / 'src' / 'initools' in result.files_created
 
 
+def test_requirement_file_options(script, data, tmpdir):
+    """
+    Test setting --install-option and --global-option in requirements files.
+    """
+
+    # When the setuppyargs package is installed, it will write the
+    # arguments with which its setup.py file was called in
+    # PIPTEST_SETUPPYARGS_FILE.
+    setuppyargs_file = tmpdir / 'setup-py-args'
+    script.environ['PIPTEST_SETUPPYARGS_FILE'] = str(setuppyargs_file)
+
+    def getsetuppyargs(contents):
+        contents = textwrap.dedent(contents)
+        script.scratch_path.join('reqfileopts.txt').write(contents)
+        script.pip('install', '--no-index', '-f', data.find_links,
+                   '-r', script.scratch_path / 'reqfileopts.txt',
+                   expect_error=True)
+        return json.load(open(setuppyargs_file))
+
+    reqfile = '''
+    setuppyargs==1.0 --global-option="--one --two" \\
```

Comment:
I don't think we should be testing `--global-option="--one --two`.  that's an invalid use of the option